compiler/src/iree/compiler/Codegen/Dialect/UKernelOps.cpp

```diff
@@ -55,12 +56,15 @@ static FailureOr<func::CallOp> createFunctionCall(
     for (auto attr : fnDefAttrs) {
       fnDecl->setAttr(attr.getName(), attr.getValue());
     }
-    // TODO(#12327): Based on description in the issue, add an attribute
-    // `vm.import.module` and set it to `vmvx`. This only works on `vmvx`
-    // backend (obviously), but is enough to unblock while the proper fix lands.
-    // For now there are a bunch of attributes set on the function, but this
-    // should be made more controllable based on the backend.
-    fnDecl->setAttr("vm.import.module", rewriter.getStringAttr("vmvx"));
+    auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(op);
+    if (isVMVXBackend(targetAttr)) {
```

Comment:
If this is needed only for VMVX, then lets move setting this attribute also to `LLVMCPULowerToUKernels.cpp`....